### PR TITLE
Chinese translation for ambari integration. 

### PR DESCRIPTION
**Pull request by lukairui**
Related to Issue [#6535](https://github.com/apache/dolphinscheduler/issues/6535)

Chinese translation for ambari integration.

### CODE REVIEW

**Comment by zhongjiajie** [comment]
Hi @lukairui, thanks for contributing.
When we add some documentation, we should also add it to dev directory, because dev directory keep our laster docs, and we would copy-paste it to new directory when we next versions dolphinscheduler release.
You could find dev directory in  https://github.com/apache/dolphinscheduler-website/tree/master/docs/zh-cn/dev/user_doc

File: site_config/docs2-0-0.js [hunk 1]
```diff
@@ -488,6 +488,15 @@ export default {
           },
         ],
       },
+      {
+        title: 'Integration',
```
Comment (zhongjiajie) [inline site_config/docs2-0-0.js:492]: Maybe we should use this in Chinese
```suggestion
        title: '集成',
```

**Comment by lukairui** [comment]
OK , I have fixed it .

**Review by zhongjiajie — APPROVED** [review]
LGTM, cc: @CalvinKirs @lenboo

**Pull request closed without merging**